engine/src/main/java/org/terasology/rendering/dag/nodes/BlurredAmbientOcclusionNode.java

```diff
@@ -54,7 +54,6 @@
     public static final ResourceUrn SSAO_BLURRED_FBO = new ResourceUrn("engine:ssaoBlurred");
     private static final ResourceUrn SSAO_FBO = new ResourceUrn("engine:ssao");
```

Comment:
Should we perhaps rename this as SSAO_BLURRED_URN and SSAO_URN? They are not FBOs! And please also check all other URNs elsewhere with the _FBO suffix.